homeassistant/components/comelit/__init__.py

```diff
@@ -23,6 +27,22 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     return True
 
 
+async def async_migrate_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Migrate old entry."""
+    _LOGGER.debug("Migrating from version %s", entry.version)
+
+    if entry.version == 1:
+        new_data = entry.data.copy()
+        new_data.update({CONF_PORT: DEFAULT_PORT, CONF_DEVICE: BRIDGE})
+
+        entry.version = 2
+        hass.config_entries.async_update_entry(entry, data=new_data)
+
+    _LOGGER.info("Migration to version %s successful", entry.version)
```

Comment:
This shouldn't be info?

Reply:
I just took the code from `shelly` integration. Anyway I completely removed `ConfigEntry` migration so.